docs/components/address.md

```diff
@@ -0,0 +1,51 @@
+# matestack core component: Address
+
+Show [specs](../../spec/usage/components/address_spec.rb)
+
+The HTML `<address>` tag implemented in ruby.
+
+## Parameters
+
+This component can take 2 optional configuration params and either yield content or display what gets passed to the `text` configuration param.
+
+#### # id (optional)
+Expects a string with all ids the address should have.
+
+#### # class (optional)
+Expects a string with all classes the address should have.
+
```

Comment:
`text (optional)` missing as input?

Reply:
@pascalwengerter any updates on this? maybe fix yourself? :)